spring-cloud-aws-dynamodb/src/main/java/io/awspring/cloud/dynamodb/DynamoDbOperations.java

```diff
@@ -37,6 +40,17 @@ public interface DynamoDbOperations {
 	 */
 	<T> T save(T entity);
 
+	/**
+	 * Saves an item in DynamoDB using the provided PutItemEnhancedRequest.
+	 *
+	 * @param putItemEnhancedRequest the request object containing the item to be saved
+	 * @param clazz the class type of the item to be saved so
+	 *     {@link software.amazon.awssdk.enhanced.dynamodb.TableSchema} can be generated.
+	 * 
+	 * @see PutItemEnhancedRequest
+	 */
+	<T> void save(PutItemEnhancedRequest<T> putItemEnhancedRequest, Class<T> clazz);
```

Comment:
[!Note] 

Here I opted to make this method void as `DynamoDbTable.putItem` does not return updated entity. There is possibility if someone is using (extensions)[https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/ddb-en-client-extensions.html] then the entity would have been update at db layer and hence passed entity would not hold latest state. 

This deviates from set norm, so happy to adapt to maintain consistency. What's your recommendation ?